shared/common-adapters/standard-screen.desktop.js

```diff
@@ -0,0 +1,67 @@
+// @flow
+import React from 'react'
+import {Box, Text, Icon} from '../common-adapters'
+import {globalStyles, globalColors, globalMargins} from '../styles/style-guide'
+import type {Props} from './standard-screen'
+
+const defaults = {
+  showClose: true,
```

Comment:
Whoops, I cut this property in a commit but forgot to remove it here. I'll dd it.